static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js

```diff
@@ -1,40 +1,42 @@
-import raven from 'lib/raven';
+
 import config from 'lib/config';
+import { getCookie } from 'lib/cookies';
 import fastdom from 'lib/fastdom-promise';
+import { loadScript } from 'lib/load-script';
+import raven from 'lib/raven';
 import sha1 from 'lib/sha1';
 import { session } from 'lib/storage';
-import { getUserFromCookie } from 'common/modules/identity/api';
-import { loadScript } from 'lib/load-script';
-import { commercialFeatures } from 'common/modules/commercial/commercial-features';
+
 import { buildPageTargeting } from 'common/modules/commercial/build-page-targeting';
+import { commercialFeatures } from 'common/modules/commercial/commercial-features';
+
+import { adFreeSlotRemove } from 'commercial/modules/close-disabled-slots';
 import { dfpEnv } from 'commercial/modules/dfp/dfp-env';
-import { onSlotRender } from 'commercial/modules/dfp/on-slot-render';
+import { fillAdvertSlots } from 'commercial/modules/dfp/fill-advert-slots';
+import { getUserFromCookie } from 'common/modules/identity/api';
 import { onSlotLoad } from 'commercial/modules/dfp/on-slot-load';
+import { onSlotRender } from 'commercial/modules/dfp/on-slot-render';
 import { onSlotViewableFunction } from 'commercial/modules/dfp/on-slot-viewable';
 import { onSlotVisibilityChanged } from 'commercial/modules/dfp/on-slot-visibility-changed';
-import { fillAdvertSlots } from 'commercial/modules/dfp/fill-advert-slots';
-import { refreshOnResize } from 'commercial/modules/dfp/refresh-on-resize';
-import { adFreeSlotRemove } from 'commercial/modules/close-disabled-slots';
 import {
     addTag,
     setListeners,
 } from 'commercial/modules/dfp/performance-logging';
-
+import { refreshOnResize } from 'commercial/modules/dfp/refresh-on-resize';
 import { init as initMessenger } from 'commercial/modules/messenger';
-
-import { init as type } from 'commercial/modules/messenger/type';
-import { init as getStyles } from 'commercial/modules/messenger/get-stylesheet';
+import { init as background } from 'commercial/modules/messenger/background';
+import { init as sendClick } from 'commercial/modules/messenger/click';
+import { init as disableRefresh } from 'commercial/modules/messenger/disable-refresh';
 import { init as getPageTargeting } from 'commercial/modules/messenger/get-page-targeting';
+import { init as getStyles } from 'commercial/modules/messenger/get-stylesheet';
 import { init as hide } from 'commercial/modules/messenger/hide';
 import { init as resize } from 'commercial/modules/messenger/resize';
 import { init as scroll } from 'commercial/modules/messenger/scroll';
+import { init as type } from 'commercial/modules/messenger/type';
 import { init as viewport } from 'commercial/modules/messenger/viewport';
-import { init as sendClick } from 'commercial/modules/messenger/click';
-import { init as background } from 'commercial/modules/messenger/background';
-import { init as disableRefresh } from 'commercial/modules/messenger/disable-refresh';
```

Comment:
what happened in this whole import list? 🙀

Reply:
Just a bit of moving around